pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -765,6 +767,23 @@ func (c *Cluster) getActivatePVCInitContainer(osdProps osdProperties, osdID stri
 	return container
 }
 
+func (c *Cluster) getExpandPVCInitContainer(osdProps osdProperties, osdID string) v1.Container {
+	osdDataPath := activateOSDMountPath + osdID
+
+	container := v1.Container{
+		Name:  expandPVCOSDInitContainer,
+		Image: c.cephVersion.Image,
+		Command: []string{
+			"ceph-bluestore-tool",
+		},
+		Args:            []string{"bluefs-bdev-expand", "--path", osdDataPath},
```

Comment:
Is this a fast command if the size didn't change? Just wondering if the startup time of the OSD will be impacted since it will run every time the OSD restarts.

Reply:
Or instead of the init container, is it possible to connect to the running OSD pod and execute the command? Or does the OSD have to be stopped to run it?

Reply:
The OSD process cannot be started. The pod takes like 1sec to run.